components/dashboard/src/admin/ProjectsSearch.tsx

```diff
@@ -0,0 +1,132 @@
+                    const owner = await getGitpodService().server.adminGetTeamById(currentProject.teamId);
+                    if (owner) { setCurrentProjectOwner(owner?.name) }
+                }
+            }
+        })();
+    }, [currentProject])
+
+    if (!user || !user?.rolesOrPermissions?.includes('admin')) {
+        return <Redirect to="/" />
+    }
+
+    if (currentProject) {
+        return <ProjectDetail project={currentProject} owner={currentProjectOwner} />;
+    }
+
+    const search = async () => {
+        setSearching(true);
+        try {
+            const result = await getGitpodService().server.adminGetProjectsBySearchTerm({
+                searchTerm,
+                limit: 50,
+                orderBy: 'creationTime',
+                offset: 0,
+                orderDir: "desc"
+            })
+            setSearchResult(result);
+        } finally {
+            setSearching(false);
+        }
+    }
+
+    return <>
+        <div className="pt-8 flex">
+            <div className="flex justify-between w-full">
+                <div className="flex">
+                    <div className="py-4">
+                        <svg className={searching ? 'animate-spin' : ''} width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fillRule="evenodd" clipRule="evenodd" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" fill="#A8A29E" />
+                        </svg>
+                    </div>
+                    <input type="search" placeholder="Search Projects" onKeyDown={(k) => k.key === 'Enter' && search()} onChange={(v) => { setSearchTerm(v.target.value) }} />
+                </div>
+                <button disabled={searching} onClick={search}>Search</button>
+            </div>
+        </div>
+        <div className="flex flex-col space-y-2">
+            <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
+                <div className="w-4/12">Name</div>
+                <div className="w-6/12">Clone URL</div>
+                <div className="w-2/12">Created</div>
+            </div>
+            {searchResult.rows.map(project => <ProjectResultItem project={project} />)}
+        </div>
+    </>
+
+    function ProjectResultItem(p: { project: Project }) {
+        return (
+            <Link key={'pr-' + p.project.name} to={'/admin/projects/' + p.project.id} data-analytics='{"button_type":"sidebar_menu"}'>
+                <div className="rounded-xl whitespace-nowrap flex py-6 px-6 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light group">
+                    <div className="flex flex-col w-4/12 truncate">
+                        <div className="font-medium text-gray-800 dark:text-gray-100 truncate hover:text-blue-600 dark:hover:text-blue-400">{p.project.name}</div>
```

Comment:
**suggestion**: Since this was also discussed for the upcoming new workspace input modal (https://github.com/gitpod-io/gitpod/pull/7715), we could re-use the same pattern which is non-existing yet to include a small indicator on hover that describes the action of the single-click open (see screenshot below). However, feel free to ignore this comment for now, bringing this up for visibility and future iterations.

| Open workspace | View project |
|-|-|
| <img width="648" alt="Modal copy" src="https://user-images.githubusercontent.com/120486/151803389-cb77e31a-fba7-4050-b17d-bb9d70c69227.png"> | <img width="1440" alt="Projects (List)" src="https://user-images.githubusercontent.com/120486/151803397-98d428c5-82f4-46cc-abfa-892d1bc57c53.png"> |